bika/lims/browser/instrument.py

```diff
@@ -610,6 +611,58 @@ def folderitems(self):
         return items
 
 
+class InstrumentAutoImportLogsView(BikaListingView):
+    """ Logs of Auto-Imports of this instrument.
+    """
+
+    def __init__(self, context, request, **kwargs):
+        BikaListingView.__init__(self, context, request, **kwargs)
+        self.columns = {'ImportTime': {'title': _('Time'),
+                                       'sortable': False},
+                        'Interface': {'title': _('Interface'),
+                                      'sortable': False,
+                                      'attr': 'getInterface'},
+                        'ImportFile': {'title': _('Imported File'),
+                                       'sortable': False,
+                                       'attr': 'getImportedFile'},
+                        'Results': {'title': _('Results'),
+                                    'sortable': False},
+                        }
+        self.review_states = [
+            {'id': 'default',
+             'title':  _('All'),
+             'contentFilter': {},
+             'columns': ['ImportTime',
+                         'Interface',
+                         'ImportFile',
+                         'Results']
+             },
+        ]
+
+        self.show_select_column = False
+        self.show_workflow_action_buttons = False
+        self.catalog = CATALOG_AUTOIMPORTLOGS_LISTING
+        self.contentFilter = {'portal_type': 'AutoImportLog',
+                              'getInstrumentUID': self.context.UID(),
+                              'sort_on': 'Created',
+                              'sort_order': 'reverse'}
+        self.title = self.context.translate(_("Auto Import Logs of %s" %
+                                              self.context.Title()))
+
+    def folderitems(self, full_objects=False, classic=False):
+        self.portal_catalog = getToolByName(self.context,
+                                            CATALOG_AUTOIMPORTLOGS_LISTING)
```

Comment:
folderitems should be called because we want to use it with classic=False (its is True as default in bika_listing)